pom.xml~

```diff
@@ -0,0 +1,612 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This "tilde" file should have been commited/pushed ! (it's a temporary backup file of your text client)